plugins/main/public/controllers/management/components/management/configuration/log-collection/log-collection-journald.js

```diff
@@ -0,0 +1,132 @@
+  isString,
+  renderValueOrDefault,
+} from '../utils/utils';
+import { settingsListBuilder } from '../utils/builders';
+import helpLinks from './help-links';
+import { LOGCOLLECTOR_LOCALFILE_PROP, LOCALFILE_JOURNALDT_PROP } from './types';
+
+/**
+ * Returns targets array parsed in one string
+ * @param {*} item
+ * @returns string => target
+ */
+const renderTargetField = item =>
+  Array.isArray(item) ? item.join(', ') : 'agent';
+
+/**
+ * Return panels title
+ * @param {*} item => log data
+ * @returns
+ */
+const panelsLabel = item =>
+  `${item.logformat} - ${renderTargetField(item.target)}`;
+
+const mainSettings = [
+  { field: 'logformat', label: 'Log format' },
+  {
+    field: 'only-future-events',
+    label: 'Only future events',
+    render: renderValueOrNoValue,
+  },
+  {
+    field: 'filters_disabled',
+    label: 'Filters Disabled',
+    render: renderValueOrDefault('true'),
+  },
+  {
+    field: 'filters',
+    label: 'Filters',
+    columns: [
+      {
+        field: 'field',
+        name: 'Field',
+      },
+      {
+        field: 'expression',
+        name: 'Expression',
+      },
+      {
+        field: 'ignore_if_missing',
+        name: 'Ignore If Missing',
+      },
+    ],
+    info: 'The configuration filters within the same group are processed with an AND logic operator. Whereas the different filter groups are processed with an OR like logic operator.',
+  },
+];
+
+class WzConfigurationLogCollectionJournald extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { currentConfig } = this.props;
+    const items = currentConfig?.[LOGCOLLECTOR_LOCALFILE_PROP]?.[
+      LOCALFILE_JOURNALDT_PROP
+    ]
+      ? settingsListBuilder(
+          currentConfig[LOGCOLLECTOR_LOCALFILE_PROP][LOCALFILE_JOURNALDT_PROP],
+          panelsLabel,
+        )
+      : [];
+
+    return (
+      <Fragment>
+        {isString(currentConfig?.[LOGCOLLECTOR_LOCALFILE_PROP]) && (
+          <WzNoConfig
+            error={currentConfig[LOGCOLLECTOR_LOCALFILE_PROP]}
+            help={helpLinks}
+          />
+        )}
+        {!currentConfig?.[LOGCOLLECTOR_LOCALFILE_PROP]?.[
+          LOCALFILE_JOURNALDT_PROP
+        ]?.length ? (
+          <WzNoConfig error='not-present' help={helpLinks} />
+        ) : null}
+        {currentConfig?.[LOGCOLLECTOR_LOCALFILE_PROP]?.[
+          LOCALFILE_JOURNALDT_PROP
+        ]?.length > 1 ? (
+          <WzConfigurationSettingsHeader
+            title='Journald events logs'
+            description='List of journald logs that will be processed'
+            help={helpLinks}
+          >
+            <WzConfigurationListSelector
+              items={items}
+              settings={mainSettings}
+            />
+          </WzConfigurationSettingsHeader>
+        ) : null}
+        {currentConfig?.[LOGCOLLECTOR_LOCALFILE_PROP]?.[
+          LOCALFILE_JOURNALDT_PROP
+        ]?.length === 1 ? (
+          <WzConfigurationSettingsGroup
```

Comment:
In case there's only 1 log I think it would be useful to also render de Setting Header, like when there are more than 1.